packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -40,12 +40,14 @@ export class DataSyncPlugin extends GraphbackPlugin {
 
       return schema;
     }
+    const modelConflictConfigSet = new Set(Object.keys(this.config.conflictConfig.models))
```

Comment:
Does it need to be a `Set`

Reply:
I think if it was an array, we would have to search for the model name every time this loop runs